SECURITY.md

```diff
@@ -0,0 +1,21 @@
+# Reporting Security Issues
+
+We at Snyk value the security community and believe that responsible disclosure of security vulnerabilities in open source packages helps us ensure the security and privacy of the users.
+
+If you believe you have found a security vulnerability on Snyk, we encourage you to let us know right away. We will investigate all legitimate reports and do our best to quickly fix the problem. Before reporting though, please review our responsible disclosure policy, and those things that should not be reported.
```

Comment:
<label>**nitpick** </label>[non-blocking]:<subject> Is there value in also linking to the title, i.e. [responsible disclosure policy](#Responsible-Disclosure-Policy)</subject>